.github/workflows/integration-test-flex-suite.yaml

```diff
@@ -1,5 +1,5 @@
 name: Integration test CephFlexSuite
-on: [pull_request]
+on: [push, pull_request]
```

Comment:
I think we need this:

```
on:
  push:
    branches:
      - master
 ```

So that we don't run on any pushed branch.

Reply:
But keep the `pull_request` too.

Reply:
done.